sahha-android/src/main/java/sdk/sahha/android/data/manager/PermissionManagerImpl.kt

```diff
@@ -216,12 +300,15 @@ internal class PermissionManagerImpl @Inject constructor(
 
     override fun openHealthConnectSettings(context: Context) {
         val packageName = context.packageManager.getPackageInfo(context.packageName, 0).packageName
-        val intent = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
-            Intent(HealthConnectManager.ACTION_MANAGE_HEALTH_PERMISSIONS)
-                .putExtra(Intent.EXTRA_PACKAGE_NAME, packageName)
-        } else {
-            Intent(HealthConnectClient.ACTION_HEALTH_CONNECT_SETTINGS)
-        }.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+        val intent = Intent(HealthConnectClient.ACTION_HEALTH_CONNECT_SETTINGS)
```

Comment:
This change was because HealthConnect setting page launch started causing crashes out of nowhere